enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -48,24 +48,26 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
             </div>
             <div class="course-info">
               <i class="fa fa-clock-o" aria-hidden="true"></i>
-              <span>${ starts_at_text } ${ course_start_date } &nbsp;| &nbsp; ${ course_pacing } - ${ course_pace_text }</span>
+              <span>${ starts_at_text } ${ course_start_date } &nbsp;| &nbsp; ${ course_pacing }</span>
```

Comment:
I believe that those are for the sake of screenreaders so that stuff doesn't appear as all one word. I'd prefer to get a better, table-or-div-based method in place at some point, but for right now, I don't have the bandwidth to modify the template that much.